examples/lighting-app/nrfconnect/main/LightingManager.cpp

```diff
@@ -95,6 +102,12 @@ bool LightingManager::InitiateAction(Action_t aAction, int32_t aActor)
     return action_initiated;
 }
 
+void LightingManager::Level(uint8_t aLevel)
```

Comment:
Level -> SetLevel? Or maybe combine this method with LightingManager::Set and make the latter set both On/Off state and the level?

Reply:
I believe `ZCL_MOVE_TO_LEVEL_COMMAND_ID` should not touch On/Off, while `ZCL_MOVE_TO_LEVEL_WITH_ON_OFF_COMMAND_ID` should touch it. Still need to investigate whether that is already handled in the ZCL level control cluster code.